Game/Scripts/UIGameManager.cpp

```diff
@@ -92,13 +95,17 @@ void UIGameManager::MenuIsOpen()
 {
 	if (menuIsOpen)
 	{
+		manager->GetComponent<EnemiesManager>()->PauseEnemies(true);
```

Comment:
at the end of the if-else block:
```suggestion
		manager->GetComponent<EnemiesManager>()->PauseEnemies(menuIsOpen);
```